ci-bench-runner/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0.75"
-askama = "0.12.1"
+askama = { version = "0.12.1", default-features = false }
```

Comment:
Could you add a note in the PR desc about what part of the default features merited a change?

Reply:
Sure! Is the current description enough?

Reply:
Yup! That's sort of what I assumed but wanted to be sure. Thanks!